packages/core/src/clients/middleware/retry/defaultRetryDecider.ts

```diff
@@ -45,10 +45,10 @@ const TIMEOUT_ERROR_CODES = [
 ];
 
 const isThrottlingError = (statusCode?: number, errorCode?: string) =>
-	statusCode === 429 || THROTTLING_ERROR_CODES.includes(errorCode);
+	statusCode === 429 || THROTTLING_ERROR_CODES.includes(errorCode ?? '');
```

Comment:
Falling back to an empty string may produce the same result, but I think it's not logically correct. 

```suggestion
	statusCode === 429 || errorCode && THROTTLING_ERROR_CODES.includes(errorCode);
```